app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -1126,6 +1135,10 @@ public void handleResult(@NonNull StreamInfo info) {
             videoUploadDateView.setVisibility(View.GONE);
         }
 
+        if (info.getServiceId() != ServiceList.YouTube.getServiceId()) {
```

Comment:
How about checking for non-html instead of non-youtube?
Also, it would be better to move this into `prepareDescription`

Reply:
I thought about that but it was late in the evening. Thanks for the reminder